### PR TITLE
Rewrite regenerate

### PR DESCRIPTION
**Pull request by david-a-wheeler**

### CODE REVIEW

File: regenerate-website-2.sh [hunk 1]
```diff
@@ -0,0 +1,1729 @@
...
+fail () {
+  printf '%s\n' "$1" >&2
+  exit 1
+}
+
+# set -eu
+set -x
+
+if [ "$(whoami)" = 'root' ]; then
+    fail 'DO NOT run this as roo!! Execute /root/run-regenerate.sh instead!!'
```
Comment (benjub) [inline regenerate-website-2.sh:18]:
```suggestion
    fail 'DO NOT run this as root!! Execute /root/run-regenerate.sh instead!!'
```

**Comment by benjub** [comment]
Version control allows you to retrieve older versions, so no need to keep the whole old script commented out, you can simply remove it.

**Comment by david-a-wheeler** [comment]
> Version control allows you to retrieve older versions, so no need to keep the whole old script commented out, you can simply remove it.

Sure, but I'm trying to re-implement the functionality we care about AND throw away the rest. I'm using the old script commented-out to help me identify what needs doing.

I don't want to remove the old script yet, because that's the script we run daily. This way I can merge this branch without ruining anything.

**Review by tirix — APPROVED** [review]

File: regenerate-website-2.sh (hunk 1 above)
Comment (benjub) [reply regenerate-website-2.sh:18]: Maybe this was meant as a pun ? One certainly should not act as root in a jumpy way.

Anyway, if this delays the merge, you can leave it like that.

**Comment by benjub** [comment]
> > Version control allows you to retrieve older versions, so no need to keep the whole old script commented out, you can simply remove it.
> 
> Sure, but I'm trying to re-implement the functionality we care about AND throw away the rest. I'm using the old script commented-out to help me identify what needs doing.
> 
> I don't want to remove the old script yet, because that's the script we run daily. This way I can merge this branch without ruining anything.

Ok, as your prefer, and it'll be deleted later. In any case, this should not delay the merge.

**Comment by jkingdon** [comment]
Is this pull request obsolete now that #16 has been merged?

I suspect so, in which case it can be closed.

**Comment by digama0** [comment]
The contents of this PR were merged into #16 in https://github.com/metamath/metamath-website-scripts/pull/16/commits/3edd32b1693af8c4874be1188b5b7ee4152cec77.

**Pull request closed without merging**